src/Chainweb/BlockHeader/Genesis.hs

```diff
@@ -139,43 +148,51 @@ genesisBlockTarget _ _ = maxTarget
 --
 -- It holds that:
 --
--- prop> mainnet20InitialHashTarget == HashTarget . (4 *) <$> (runGet decodePowHashNat "DOordl9cgfs4ZTBdFnbjRW5th-hW-pL33DIAAAAAAAA")
+-- prop> Just mainnet20InitialHashTarget == HashTarget . (4 *) <$> (runGet decodePowHashNat =<< decodeB64UrlNoPaddingText "DOordl9cgfs4ZTBdFnbjRW5th-hW-pL33DIAAAAAAAA")
 --
 mainnet20InitialHashTarget :: HashTarget
-mainnet20InitialHashTarget = HashTarget 92812039490652836170182663013446053204096613861175006070293989356886611016976
+mainnet20InitialHashTarget = HashTarget 326935740379188069500505933882552768991030445202673027909658672
```

Comment:
I might suggest defining these constants in hex and padded with leading zeroes to 256 bits to make them easier to inspect by hand.

Reply:
Makes sense. I may add a test case for the hex representation, just to make sure that I don't mess up the `printf` format string :-). Or is there a way to tell ghci to print numbers in hex by default?